.github/workflows/linux-build-and-test.yaml

```diff
@@ -59,10 +60,20 @@ jobs:
         # This is needed because GitHub image macos-10.15 version
         # 20210927.1 did not install OpenSSL so we install openssl
         # explicitly.
-        brew install openssl
+        brew install openssl gawk
         sudo perl -MCPAN -e "CPAN::Shell->notest('install', 'IPC::Run')"
         sudo perl -MCPAN -e "CPAN::Shell->notest('install', 'Test::Most')"
 
+    - name: Checkout TimescaleDB
+      uses: actions/checkout@v3
+
+    # We are going to rebuild Postgres at least once a month, so that it doesn't
+    # suddenly break ages after the original problem.
+    - name: Get date for build caching
+      id: get-date
+      run: |
+        echo "date=$(date +"%m")" >> $GITHUB_OUTPUT
```

Comment:
hmm i think weekly or even daily would be better

Reply:
Changed to daily.